src/components/FunctionsPage/functions.util.js

```diff
@@ -273,7 +267,7 @@ export const generateActionsMenu = (
         label: 'View YAML',
         icon: <Yaml />,
         disabled: functionIsDeleting,
-        onClick: (funcMin) => getFullFunction(funcMin).then((func) => toggleConvertedYaml(func))
+        onClick: funcMin => getFullFunction(funcMin).then(func => toggleConvertedYaml(func))
```

Comment:
You can simplify it to: 
onClick: funcMin => getFullFunction(funcMin).then(toggleConvertedYaml)